--- blueReg_std.conf	2012-11-01 16:09:16.392845831 -0400
+++ blueReg_std.conf.new	2012-11-01 16:09:41.944911387 -0400
@@ -72,11 +72,9 @@
 dv asym_dslumi3_dslumi7_sum
 dv asym_dslumi4_dslumi8_sum
 dv asym_uslumi_sum
-dv asym_uslumi1_uslumi5_sum
 dv asym_uslumi3_uslumi7_sum
 dv asym_uslumi1_sum
 dv asym_uslumi3_sum
-dv asym_uslumi5_sum
 dv asym_uslumi7_sum
 ### solo pmts
 dv asym_qwk_dslumi1
@@ -91,8 +89,6 @@
 dv asym_qwk_uslumi1pos
 dv asym_qwk_uslumi3neg
 dv asym_qwk_uslumi3pos
-dv asym_qwk_uslumi5neg
-dv asym_qwk_uslumi5pos
 dv asym_qwk_uslumi7neg
 dv asym_qwk_uslumi7pos
 # aux adde on Geb 19, 2011
@@ -156,7 +152,6 @@
 dv asym_uslumi_detavg
 dv asym_uslumi1_avg
 dv asym_uslumi3_avg
-dv asym_uslumi5_avg
 dv asym_uslumi7_avg
 dv asym_uslumi15_dipole
 dv asym_uslumi37_dipole
